checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java

```diff
@@ -26,6 +26,7 @@ public AinferTestCheckerJaifsTest(List<File> testFiles) {
         "ainfer-testchecker/non-annotated",
         "-Anomsgtext",
         "-Ainfer=jaifs",
+        "-AskipDefs=SimpleRecord",
```

Comment:
What is the reason for `-AskipDefs` here and below?  It seems that would avoid processing the new test case that you added.

Reply:
The AFU's jaif reading/writing code doesn't support Java 17, and so executing jaif-based WPI on this test case causes a crash in the AFU.

Reply:
I'll add a comment to that effect here.